From: Chow Loong Jin <hyperair@debian.org>
Date: Mon, 12 Jun 2023 13:30:41 +0800
Subject: Remove redundant add_library(libexpat INTERFACE)

This causes the following build failure:

    CMake Error at CMakeLists.txt:449 (add_library):
      add_library cannot create target "libexpat" because another target with the
      same name already exists.  The existing target is an interface library
      created in source directory
      "/<<PKGBUILDDIR>>".  See
      documentation for policy CMP0002 for more details.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index db7c306..e84ced1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -457,7 +457,7 @@ include_directories(BEFORE SYSTEM ${EIGEN3_INCLUDE_DIR})
 # no matter what.
 find_package(EXPAT REQUIRED)
 
-add_library(libexpat INTERFACE)
+# add_library(libexpat INTERFACE)
 
 if (TARGET EXPAT::EXPAT )
     target_link_libraries(libexpat INTERFACE EXPAT::EXPAT)
